pkg/storage/etcd/testing/utils.go

```diff
@@ -164,8 +311,8 @@ func (m *EtcdTestServer) Terminate(t *testing.T) {
 }
 
 // NewEtcdTestClientServer creates a new client and server for testing
-func NewEtcdTestClientServer(t *testing.T) *EtcdTestServer {
-	server := configureTestCluster(t, "foo")
+func NewEtcdTestClientServer(t *testing.T, secure bool) *EtcdTestServer {
+	server := configureTestCluster(t, "foo", secure)
```

Comment:
@liggitt I did consider this way, but for some reason I decided to add a flag to control this, add a second method can eliminate great impacts on other test cases.